### DISCUSSION TITLE
How to solve ree-validate issue?

### DISCUSSION THREAD

**Discussion by Elessar1996** [vercel/next.js#11437] [Help]
I'm working on debugging a Nextjs project which I have downloaded from Git. When I try to install its packages I get the error below:

" ./utils/validator.js Module not found: Can't resolve 'ree-validate' in '/project/path/utils' " .

I have tried Installing the "ree-validate" separately but I still get the error above. How can I fix this issue?

**Comment by timneutkens** [comment]
Not sure what you mean exactly, can you provide more context.
